iFixit/react-commerce
# Simplify warranty banner component and adjust according to mockup

closes #1806 

## QA

1. Open a product list page that has the lifetime warranty section (e.g. [/Parts](https://react-commerce-git-refactor-product-list-warranty-banner-ifixit.vercel.app/Parts))
2. Verify that the banner renders correctly and according to [mockup](https://www.figma.com/file/BolmFORO6ZfHXePOSyEGSS/iFixit---Page-sections?type=design&node-id=383-9494&mode=dev)

![Screenshot 2023-07-06 at 17 10 16](https://github.com/iFixit/react-commerce/assets/4640135/8e445ac7-9baf-4124-9888-723f6de4aef4)
